### DISCUSSION TITLE
Will there be a table functionality in the editor?

### DISCUSSION THREAD

**Discussion by langerant** [slab/quill#4240] [General]
Hey everyone!
I am currently looking for a rich text editor, which I can use in my PWA. A friend recommended the Quill editor. The only downside of this editor is that in my PWA people should have the possibility to write comments and within this a usecase would be to create tables (see image). 
Is something like this planned in the near future?

![nS1Z8S0F9s](https://github.com/slab/quill/assets/148343655/488dcf2f-e626-4a8a-8e01-9a01399d7eca)

**Comment by malno-dev** [comment]
@langerant I'm looking for a rich text editor for the same use case you listed. I'm hoping tables are being considered.

**Comment by jgboughey** [comment]
Did anyone receive a response whether "Tables" would be considered as a new feature for the Quil text editor?

**Comment by artknight** [comment]
try https://github.com/zzxming/quill-table-up. it's pretty damn good!
